releasenotes/notes/runtime-provider-support-5358b72ec0035419.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Experiments run via the ``qiskit-ibm-runtime`` provider can now be saved
+    to the cloud service.
+upgrade:
+  - |
+    Support for ``qiskit-ibm-provider`` is now deprecated and will be removed
```

Comment:
I would phrase this in a way that points out that  qiskit-ibm-provider is itself deprecated and replaced by qiskit-ibm-runtime rather than qiskit-experiments choosing not to support it.